lib/core/diagnostics.js

```diff
@@ -114,6 +120,76 @@ if (undiciDebugLog.enabled || fetchDebuglog.enabled) {
   isClientSet = true
 }
 
+// Track fetch requests
+if (fetchDebuglog.enabled) {
+  const debuglog = fetchDebuglog.enabled
+
+  diagnosticsChannel.channel('undici:fetch:start').subscribe(evt => {
+    const {
+      input
+    } = evt
+    debuglog(
+      'fetch request has started request to %s',
+      input
+    )
+  })
+
+  diagnosticsChannel.channel('undici:fetch:end').subscribe(evt => {
+    const {
+      input,
+      error
+    } = evt
+    debuglog(
+      'received response from %s, error %s',
+      input,
+      error?.message
+    )
+  })
```

Comment:
What I dont like is , that if we dont have an  error, it will output "received response from google.com, error undefined"

Reply:
@Uzlopak I have removed the error portion. You are correct I would not expect an error in this case, that would be handled by the error event.